tests/roots/test-ext-autodoc/target/typehints.py

```diff
@@ -2,9 +2,25 @@ def incr(a: int, b: int = 1) -> int:
     return a + b
 
 
+def decr(a, b = 1):
+    # type: (int, int) -> int
+    return a - b
+
+
 class Math:
     def __init__(self, s: str, o: object = None) -> None:
         pass
 
     def incr(self, a: int, b: int = 1) -> int:
         return a + b
+
+    def decr(self, a, b = 1):
+        # type: (int, int) -> int
+        return a - b
+
+
+def complex_func(arg1, arg2, arg3=None, *args, **kwargs):
+    # type: (str, List[int], Tuple[int, Union[str, Unknown]], str, str) -> None
```

Comment:
According to [this post](https://stackoverflow.com/a/38066950/102441)
```suggestion
    # type: (str, List[int], Tuple[int, Union[str, Unknown]], *str, **str) -> None
```

Can you add a test for `Literal` too?